Makefile

```diff
@@ -111,6 +115,21 @@ static-test:
 .PHONY: test
 test: unit-test static-test
 
+.PHONY: tarball
+tarball:
```

Comment:
should this depend on some other target, like build? otherwise the assets will not be there...or I'm missing something?

Reply:
yes, this makes sense, will fix this